### ISSUE TITLE
[Feature request] Add the ability to use a separate pacman.conf

### ISSUE THREAD

**Issue by Am0g-us** [vn971/rua#214]
This feature will be useful for those who use distributions like cachyos, where some aur packages are built with a version higher than in aur (e.g. zfs-utils-2.1.12-2 in cachyos and zfs-utils-2.1.12-1 in aur), which causes pacman to update them from the cachyos repository, which can be a problem if the user's processor supports x86_64v4. So I suggest to add the ability to set the list of ignored packages in a separate file or a separate pacman.conf.

**Comment by vn971** [comment]
Hi @Am0g-us, if you're talking about `rua upgrade`, would an `--ignore` flag work for your use case?

```log
rua upgrade --help
...
...
OPTIONS:
        --ignore <ignored>    Don't upgrade the specified package(s). Accepts multiple arguments separated by `,`.
```

**Comment by Am0g-us** [comment]
No. Just the opposite. Add the ability to update packages by ignoring IgnorePkg in pacman.conf. I think it is best to add a separate pacman.conf in ~/.config/rua/.system/

**Comment by vn971** [comment]
@Am0g-us I've taken a look. If you add a CLI option to specify an alternative pacman.conf location (`Option<Path>`) and use this value when calling `pacman-conf` (pacman.rs:13), I'll merge this in.
Otherwise, I might return to it a bit later when I have more time (or anyone else interested in the feature can)

**Comment by vn971** [comment]
I've also added a `good first issue` label now (because it is an easy/good first issue)
